test/karma.conf.js

```diff
@@ -74,6 +74,7 @@ module.exports = function (config) {
       'lib/modernizr.custom.80690.js',
       // application files
       'app/**/*.js',
+      'app/home/**/*.js',
```

Comment:
Well, it's supposed to be all included on 'app/**/*.js',
But the moduels where not leading either way 
so  'app/home/*.js', and 'app/coomon/*.js', helped.
I think there is a problem in the order/priority